app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="flag_width">110dp</dimen>
+    <dimen name="flag_height">110dp</dimen>
```

Comment:
as flag's width and height is the same, I guess we can have one instead, maybe we call it `flag_size`